contrib/tablefunc/tablefunc--1.0--1.1.sql

```diff
@@ -0,0 +1,15 @@
+/* contrib/tablefunc/tablefunc--1.0--1.1.sql */
+
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION tablefunc UPDATE TO '1.1'" to load this file. \quit
+
+ALTER function crosstab(text) EXECUTE ON INITPLAN;
+ALTER function crosstab2(text) EXECUTE ON INITPLAN;
+ALTER function crosstab3(text) EXECUTE ON INITPLAN;
+ALTER function crosstab4(text) EXECUTE ON INITPLAN;
+ALTER function crosstab(text,integer) EXECUTE ON INITPLAN;
```

Comment:
Isn't this one missing in the test?

Reply:
That one is declared obsolete and isn't tested even in already existing part of the test. I think we don't need to test it